httpserver-rs/Cargo.toml

```diff
@@ -1,33 +1,36 @@
 [package]
 name = "wasmcloud-provider-httpserver"
-version = "0.17.0"
+version = "0.17.1"
 edition = "2021"
 
 [dependencies]
 async-trait = "0.1.52"
 atty = "0.2"
-base64 = "0.13"
+base64 = "0.21"
 bytes = "1.2"
+dashmap = "5.4.0"
+flume = "0.10.14"
 futures = "0.3"
 http = "0.2"
 opentelemetry = { version = "0.18.0", features = ["rt-tokio"] }
 serde_bytes = "0.11"
 serde_json = "1.0"
 serde = {version = "1.0", features = ["derive"] }
+smithy-bindgen = "0.1"
 thiserror = "1.0"
 tokio = { version = "1", features = ["full"] }
-toml = "0.5"
+toml = "0.7"
 tracing = "0.1.34"
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 warp = { version="0.3", features=["tls"] }
-wasmcloud-interface-httpserver = "0.8.1"
-wasmbus-rpc = { version = "0.11.2", features = ["otel"] }
+wasmbus-rpc = { version = "0.12.0", features = ["otel"]}
+
 
 [dev-dependencies]
 assert_matches = "1.5"
 blake2 = "0.10.4"
 reqwest = { version = "0.11", features = ["json"]}
-wasmcloud-test-util = "0.6.4"
+wasmcloud-test-util = { git = "https://github.com/wasmcloud/wasmcloud-test", branch = "fix/bindgen" }
```

Comment:
reminder to return to version after that PR merges

Reply:
👍 still waiting

Reply:
done!